client/src/components/team.js

```diff
@@ -1,30 +1,52 @@
 import React from "react";
+import PropTypes from 'prop-types';
 import Player from "./player";
 
 function Team(props) {
-  const use_solo_stats = props.team.length === 1;
+  const stats = props.team.length === 1 ? 'solo_stats' : 'team_stats';
+
   return (
     <React.Fragment>
       {props.team.map(player => (
         <Player key={player.id}
-          {...{
-            name: player.name,
-            alias: player.alias,
-            race: player.race,
-            wins: use_solo_stats
-              ? player.solo_stats.wins
-              : player.team_stats.wins,
-            losses: use_solo_stats
-              ? player.solo_stats.losses
-              : player.team_stats.losses,
-            win_percentage: use_solo_stats
-              ? player.solo_stats.win_percent
-              : player.team_stats.win_percent
-          }}
+            name={player.name}
```

Comment:
all the same just seems more readable and extendable in case we add ffa support in future